hoomd/hpmc/UpdaterMuVT.h

```diff
@@ -1093,7 +1093,8 @@ template<class Shape> void UpdaterMuVT<Shape>::update(uint64_t timestep)
 
                 if (nonzero)
                     {
-                    lnboltzmann += lnb;
+                    Scalar kT = m_mc->getTimestepkT(timestep);
```

Comment:
```suggestion
                    const Scalar kT = (*m_mc->getKT())(timestep);
```
Move this line to the top of the function. `kT` does not need to be reevaluated in the inner loops.